bin/plot_krun_results

```diff
@@ -384,26 +529,29 @@ def draw_page(is_interactive, executions, cycles_executions, titles, window_size
     y_min -= adj
     y_max += adj
 
-    fig, axes = plt.subplots(n_rows, n_cols, squeeze=False)
+    fig = pyplot.figure()
+    outer_grid = gridspec.GridSpec(n_rows, n_cols)
 
     index, row, col = 0, 0, 0
     while index < n_execs:
         data = executions[index]
-        cycles_data = cycles_executions[index]
+        cycles_data = cycles_executions[index] if cycles_executions is not None else None
```

Comment:
Let's please not use Python's horrible ternary operator -- it leads to buggy code.